src/uploaders/NodeUploader.ts

```diff
@@ -223,7 +223,7 @@ export async function uploadMultiple ({
         type: PayloadType.Node,
         apiKey,
         appVersion,
-        minifiedUrl: path.relative(projectRoot, path.resolve(absoluteSearchPath, bundlePath)),
+        minifiedUrl: path.relative(projectRoot, path.resolve(absoluteSearchPath, bundlePath)).replace(/\\/g, '/'),
```

Comment:
does it make sense to try to normalise the file paths before getting to this stage? For instance, `directory`, `projectRoot` etc. so that all of the path operations (`dirname`, `resolve` etc.) aren't run on mixed input – some contain `\` and some containing `/`?

Reply:
I don't think that would work as `path.resolve` (and other `path` functions) will always use `\` on Windows, e.g.

```js
> path.resolve('./build/index.js')
'C:\\path\\to\\project\\build\\index.js'
```

Reply:
so does `path.resolve` _just work™️_ when you give it two paths with different separators?

Reply:
Yeah, all of these give the same output on Windows:

- `path.resolve('./build/index.js')`
- `path.resolve('.\\build/index.js')`
- `path.resolve('.\\build\\index.js')`